### ISSUE TITLE
Changing the window size does not effect the waitress line

### ISSUE THREAD

**Issue by Dschaykib** [JohnCoene/waiter#84]
Hi,
when changing the window size off an app while a waitress "line" call is running, it does not change the width of the line.

**Comment by JohnCoene** [comment]
Indeed, thank you for reporting this. It's from the JavaScript library the waitress depends on, I'm not sure I'll be able to fix but I'll try.
